docs/assignment1.md

```diff
@@ -1,62 +1,73 @@
+
+   				   Figure 1: A use case diagram of VUORK
+
+![VuORK use case diagram](https://i.postimg.cc/SRBV1RM5/Screen-Shot-2020-03-01-at-6-40-11-PM.png)
+
+The remainder of this paper will cover detailed features and quality requirements to help us structure this project (This should be self explanatory). 
 
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+**NB**: THE SYSTEM WILL BE INDEPENDENT FROM THE STORY LINE OUTLINED ABOVE. The idea is to create a system that can be *easily* adapted to any other storyline and easy to expand. 
+
+## Features
+Author(s): Marta Jansone, Theresa Schantz, Eduardo Lira, Irene Garcia-Fortea Garcia, Florent Brunet de Rochebrune 
 
 ### Functional features
 
 As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
 
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+| ID   | Short name | Description                                                  |
+| ---- | ---------- | :----------------------------------------------------------- |
+| F1   | Commands   | The player will interact with the game using the command line interface. It is, thus, intrisically important to have a parser that will understand some simple semantics of langauge. For example, a command such as *"north"* should be equivilant to *walk north*. <br> After careful consideration, we have decided that our parser must have the input in the form of *<action\> <item(s)/objects(s) [proposition]...\*>*. An item may be preceeded by a proposition. For example, *"Put cheese in fridge"* or *"Attack bear with knife"*. The action put, will let the parser know that there will be 2 items, interupted by some proposition. Other actions, however, may exclude a propostion. An example is *"Take broom"* or *"Move north"*.<br>Emposing this structure will make it possible to parse natural language to something our system can understand. If the user, for any reason, excludes a second item in a 2-part command (such as attack), then the system will reply with *"What do you want to attack the <object\> with?"*, if the item is ommited. Or *"What do you want to attack?"*, if an object is omitted. If an invalid propositon is used, for example *"Attack monkey on knife"*, the parser should reply, *"I do not understand how to Attack monkey on knife"*<br> |
```

Comment:
{feedback from assignment 1 not properly addressed} It is still unclear what are the possible actions described by the commands. Try defining precisely what are the available commands, what happens when they are invoked and how the game is affected by them. Try making them simple as simple as possible.

Reply:
thank you for your feedback. However, as stated in the documentation, we did not give a list of possible actions because this should be easily created by a 'creater' who can parse his/her JSON file with Areas, and Items. Each Item would later be defined to have certain traits (also in the JSON files)

Reply:
Hi edward. As I stated on my review, from the descriptions provided is very difficult to get an idea of the overall intention of the project, the game structure, objectives, etc. From that it was also hard mapping the requirements to the models. If that is the case, please provide a more detailed explanation about the realtionship between the creator and the game mechanics